service/nodeConnectivityChecker.go

```diff
@@ -98,7 +98,13 @@ func (s *service) apiRouter(ctx context.Context) {
 	router.HandleFunc(arrayStatus, connectivityStatus).Methods("GET")
 	router.HandleFunc(arrayStatus+"/"+"{arrayId}", getArrayConnectivityStatus).Methods("GET")
 	//start http server to serve requests
-	err := http.ListenAndServe(apiPort, router)
+	server := &http.Server{
```

Comment:
I dont think this should show up in this PR if branch was rebased with main.

Reply:
Updated. Rebased with main.